R/axis_combmatrix.R

```diff
@@ -254,10 +254,21 @@ make_combination_matrix_plot <- function(labels, labels_split, label_set, range,
   if(isTRUE(theme$combmatrix.panel.striped_background)){
     plt <- plt + geom_rect(aes(fill= .data$index %% 2 == 0), ymin=df2$index-0.5, ymax=df2$index+0.5, xmin=0, xmax=1)
   }
+  if (!isTRUE(theme$combmatrix.panel.line.size == 0)) {
```

Comment:
This empty if statement could be deleted, right?
If yes, please do so :)

Reply:
Yes! Sorry about that.